src/web_interface/rest/rest_firmware.py

```diff
@@ -29,7 +29,7 @@
     'file_name':  fields.String(description='File Name', required=True),
     'version':  fields.String(description='Version', required=True),
     'vendor':  fields.String(description='Vendor', required=True),
-    'release_date':  fields.String(description='Release Date'),
+    'release_date':  fields.String(description='Release Date', default='1970-01-01'),
```

Comment:
This works okay. But as we concluded, we should also validate the passed date string upon retrieval to keep the data nice and tidy

Reply:
added validation

Reply:
Sadly, the validation breaks the [test_rest_upload_valid](https://github.com/fkie-cad/FACT_core/blob/9da01ce172a7b817c9ab23aeb90afe3870f1e7eb/src/test/integration/web_interface/rest/test_rest_firmware.py#L61-L76) test, which now provides an invalid date value.

Reply:
should be fixed now